### ISSUE TITLE
Expanded content about Windows support

### ISSUE THREAD

**Issue by legowerewolf** [passkeydeveloper/passkeys.dev#121]
### Summary

At present, the docs just say that passkey implementation for windows is "planned" for local authenticators and "partially supported" for external authenticators. 

It would be nice to see references to where MS has announced planned support, and notes about what exactly partial support means. 

### Proposed Content Type

Other / Unknown

### Proposed Content Title

N/A: add to existing article

### Related Articles

- https://passkeys.dev/docs/reference/windows/

**Comment by timcappalli** [comment]
I'm not aware of any formal public announcement, so unfortunately there is nothing to link to.

RE: partially supported, it is outline at https://passkeys.dev/device-support/

**Comment by timcappalli** [comment]
@legowerewolf I added some additional text from /device-support/ to the Windows reference page in [6ffca66](https://github.com/passkeydeveloper/passkeys.dev/commit/6ffca669bea01c5f72433526718ef37d79c69cf8). I hope that helps explain things clearer

**Comment by legowerewolf** [comment]
Yeah, that helps. Is this the joint FIDO announcement you mentioned? 

https://fidoalliance.org/apple-google-and-microsoft-commit-to-expanded-support-for-fido-standard-to-accelerate-availability-of-passwordless-sign-ins/

**Comment by timcappalli** [comment]
> Is this the joint FIDO announcement you mentioned?

Yes, that's it

**Comment by timcappalli** [comment]
Added clarity
